torch/_inductor/codecache.py

```diff
@@ -2019,7 +2019,9 @@ def _compile_consts_darwin(consts: bytes) -> str:
 
             output_o = os.path.splitext(input_path)[0] + ".o"
             consts_size = sum(
-                tensor.untyped_storage().nbytes()
+                torch.ops.mkldnn._data_size(tensor)
```

Comment:
Does `_data_size` return nbytes? If yes, why not naming is `nbytes`?

Reply:
Yes it returns nbytes. Renamed as suggested.